src/olympia/search/filters.py

```diff
@@ -1011,14 +1011,15 @@ def filter_queryset(self, request, qs, view):
                     or AddonPromotedQueryParam.query_param in request.GET
                 ) and not search_query_param
                 if is_random_sort_available:
-                    # We want randomness to change only once every 24 hours, so
-                    # we use a seed that depends on the date.
+                    # Third, we want randomness to change only once every hour,
+                    # so we use a seed that depends on the date + hour.
+                    now = datetime.now()
                     qs = qs.query(
                         'function_score',
                         functions=[
                             query.SF(
                                 'random_score',
-                                seed=date.today().toordinal(),
+                                seed=now.toordinal() * 24 + now.hour,
```

Comment:
Just pointing out that the greater the frequency of the seed changing, the more chance that a user will be browsing a list of search results sorted by random and the order suddenly changes when they're on x page of y results.

Reply:
I agree, I'm following what was asked though - we can revisit if this cause problems. I think this is still a long enough period of time that it's not a huge issue, and also the recommended shelf (only search that is likely to be using random sort) is small enough that users won't be paginated through the results a lot.